Possible regression in 1.0.64 and #190?

#190 seems to fix #185, video does stop, but we now get this error:

```
Uncaught exception thrown by finalizer
java.lang.AssertionError: VLCObject (org.videolan.libvlc.Media) finalized but not natively released (1 refs)
     at org.videolan.libvlc.VLCObject.finalize(VLCObject.java:101)
     at java.lang.Daemons$FinalizerDaemon.doFinalize(Daemons.java:319)
     at java.lang.Daemons$FinalizerDaemon.runInternal(Daemons.java:306)
     at java.lang.Daemons$Daemon.run(Daemons.java:140)
     at java.lang.Thread.run(Thread.java:1012)
```
@anudeepreddy may be there is some additional condition missing?

--- a/android/src/main/java/com/yuanzhou/vlc/vlcplayer/ReactVlcPlayerView.java
+++ b/android/src/main/java/com/yuanzhou/vlc/vlcplayer/ReactVlcPlayerView.java
@@ -399,6 +399,7 @@ class ReactVlcPlayerView extends TextureView implements
             }
             mVideoInfo = null;
             mMediaPlayer.setMedia(m);
+            m.release();
             mMediaPlayer.setScale(0);
             if (_subtitleUri != null) {
                 mMediaPlayer.addSlave(Media.Slave.Type.Subtitle, _subtitleUri, true);

